packages/datetime/src/datePickerCore.tsx

```diff
@@ -50,6 +50,11 @@ export interface IDatePickerBaseProps {
      */
     dayPickerProps?: DayPickerProps;
 
+    /**
+     * An additional component to show below the date picker
+     */
+    footerComponent?: JSX.Element;
```

Comment:
this isn't a component, it's an element:

```suggestion
    footerElement?: JSX.Element;
```

comment should be updated to match